router/api/router.go

```diff
@@ -54,6 +52,11 @@ type routesReq struct {
 	Addresses []string `json:"addresses"`
 }
 
+type swapReq struct {
+	Target    string
```

Comment:
Other structs here add tags to make json fields lowercase, shouldn't we do the same thing here?

Reply:
done